com.github.opentyrian.OpenTyrian.yaml

```diff
@@ -0,0 +1,56 @@
+app-id: com.github.opentyrian.OpenTyrian
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+finish-args:
+  - --device=all
+  - --share=network
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Pulseaudio access
+  - --socket=pulseaudio
+  # OpenGL access
+  - --device=dri
+  #SDL needs to be able to talk to UPower
+  - --system-talk-name=org.freedesktop.UPower
+  #Allow SDL to inhibit the screensaver
+  - --talk-name=org.freedesktop.ScreenSaver
+command: opentyrian
+rename-appdata-file: opentyrian.appdata.xml
+rename-desktop-file: opentyrian.desktop
+rename-icon: opentyrian
+copy-icon: true
+modules:
+  - name: opentyrian
+    buildsystem: simple
+    build-commands:
+      - make install DESTDIR=${FLATPAK_DEST}/ prefix= gamesdir=/share/
```

Comment:
```suggestion
      - make install DESTDIR=${FLATPAK_DEST}/ prefix= gamesdir=/extra
```

Reply:
you should be using prefix and NOT `DESTDIR`. `DESTDIR` is for staging.